programs/create-genesis/genesis_create.cpp

```diff
@@ -101,6 +99,18 @@ struct genesis_create::genesis_create_impl final {
         return name_by_idx(_visitor.acc_id2idx[id]);
     }
 
+    supply_distributor get_gbg_supply_distributor() {
```

Comment:
in this case it doesn't distribute supply, but just converts, propose to rename this function to make this clear. maybe `get_gbg_to_golos_converter` or `get_gbg_converter` or so